test/TestAssets/SimpleTestProject/SimpleTestProject.csproj

```diff
@@ -1,19 +1,22 @@
 ﻿<?xml version="1.0" encoding="utf-16"?>
 <Project Sdk="Microsoft.NET.Sdk">
+  <!-- Package dependency versions -->
+  <Import Project="..\..\..\scripts\build\TestPlatform.Dependencies.props" />
+
   <PropertyGroup>
     <TargetFrameworks>netcoreapp1.0;net46;netcoreapp1.1</TargetFrameworks>
     <AssemblyName>SimpleTestProject</AssemblyName>
     <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netcoreapp1.0' Or '$(TargetFramework)' == 'netcoreapp1.1' ">$(PackageTargetFallback);dnxcore50;portable-net45+win8</PackageTargetFallback>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="MSTest.TestFramework">
-      <Version>1.0.6-preview</Version>
+      <Version>$(MSTestFrameworkVersion)</Version>
     </PackageReference>
     <PackageReference Include="MSTest.TestAdapter">
-      <Version>1.1.6-preview</Version>
+      <Version>$(MSTestAdapterVersion)</Version>
```

Comment:
This should be sync with  [acceptance test reference](https://github.com/Microsoft/vstest/blob/master/test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs#L31).
Is there way we can read reference from `prop` file directly in acceptance or we have to sync both manually always?